include/Core/Metas/Modules/DeleteTermMetaModule.php

```diff
@@ -0,0 +1,176 @@
+<?php
+namespace BulkWP\BulkDelete\Core\Metas\Modules;
+
+use BulkWP\BulkDelete\Core\Metas\MetasModule;
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Delete Term Meta.
+ *
+ * @since 6.1.0
+ */
+class DeleteTermMetaModule extends MetasModule {
+	/**
+	 * Initialize the Module.
+	 */
+	protected function initialize() {
+		$this->field_slug    = 'meta_term';
+		$this->meta_box_slug = 'bd-meta-term';
+		$this->action        = 'delete_meta_term';
+		$this->cron_hook     = 'do-bulk-delete-term-meta';
+		$this->messages      = array(
+			'box_label'  => __( 'Bulk Delete Term Meta', 'bulk-delete' ),
+			'scheduled'  => __( 'Term meta fields from the posts with the selected criteria are scheduled for deletion.', 'bulk-delete' ),
+			'cron_label' => __( 'Delete Term Meta', 'bulk-delete' ),
+		);
+	}
+
+	/**
+	 * Render the Modules.
+	 *
+	 * @return void
```

Comment:
Don't add phpdoc block for inherited methods.

Reply:
Fixed